### PR TITLE
Python 3 fixes

### PR DESCRIPTION
**Pull request by rahulporuri**

### CODE REVIEW

File: ensemble/ctf/editor.py [hunk 1]
```diff
@@ -243,7 +243,7 @@ def _draw_histogram(self, gc):
     def _add_components_for_new_function(self, function):
         linked_colors, linked_opacities = [], []
         if len(function.links) > 0:
-            linked_colors, linked_opacities = zip(*function.links)
+            linked_colors, linked_opacities = list(zip(*function.links))
```
Comment (jwiggins) [inline ensemble/ctf/editor.py]: This isn't necessary

Comment (jwiggins) [reply ensemble/ctf/editor.py]:
```
In [1]: things = [(1, 2), (3, 4)]

In [2]: zip(*things)
Out[2]: <zip at 0x1067c3408>

In [3]: left, right = zip(*things)

In [4]: left
Out[4]: (1, 3)

In [5]: right
Out[5]: (2, 4)
```

File: ensemble/ctf/manager.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 from .utils import load_ctf, save_ctf
 
 
-CTF_EXTENSION = '.ctf'
+CTF_EXTENSION = u'.ctf'
```
Comment (jwiggins) [inline ensemble/ctf/manager.py]: How do you feel about `from __future__ import unicode_literals`? I think the test coverage of this module is probably good enough to consider it.

File: ensemble/ctf/window_function_component.py [hunk 3]
```diff
@@ -260,7 +260,7 @@ def _tools_default(self):
         function = self.container.function
         items = [WindowTypeAction(name=name.capitalize(), component=self,
                                   function=function)
-                 for name in WINDOW_FUNCTIONS.keys()]
+                 for name in list(WINDOW_FUNCTIONS.keys())]
```
Comment (jwiggins) [inline ensemble/ctf/window_function_component.py]: Unnecessary

File: ensemble/ctf/window_function_component.py [hunk 4]
```diff
@@ -97,7 +97,7 @@ def values(self):
 
         xs = np.linspace(center - radius, center + radius, num_samples)
         ys = self.window_func(num_samples) * self.opacity
-        return zip(xs, ys)
+        return list(zip(xs, ys))
```
Comment (jwiggins) [inline ensemble/ctf/window_function_component.py]: Hmm... Probably needed, but it just makes me wonder about the downstream code (keeping in mind that I wrote most of said code... ahem).

File: ensemble/ctf/manager.py (hunk 2 above)
Comment (rahulporuri) [reply ensemble/ctf/manager.py]: The package itself is small enough that i'd prefer just using the `unicode_literals` import in all modules.

Comment (jwiggins) [reply ensemble/ctf/manager.py]: Agreed

**Comment by jwiggins** [comment]
Oh... Looks like `unicode_literals` _is_ a problem

**Comment by rahulporuri** [comment]
closing this in favor of #66

**Pull request closed without merging**